src/AzureExtension/DevBox/Models/CreateComputeSystemOperation.cs

```diff
@@ -0,0 +1,110 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using AzureExtension.Contracts;
+using AzureExtension.DevBox.Exceptions;
+using DevHomeAzureExtension.Helpers;
+using Microsoft.VisualStudio.Services.Commerce;
+using Microsoft.VisualStudio.Services.TestManagement.TestPlanning.WebApi;
+using Microsoft.Windows.DevHome.SDK;
+using Windows.Foundation;
+
+namespace AzureExtension.DevBox.Models;
+
+public delegate CreateComputeSystemOperation CreateComputeSystemOperationFactory(IDeveloperId developerId, DevBoxCreationParameters userOptions);
+
+/// <summary>
+/// Responsible for creating a new DevBox. It is the object return back to Dev Home so Dev Home can track the progress of the operation.
+/// </summary>
+public class CreateComputeSystemOperation : ICreateComputeSystemOperation
+{
+#pragma warning disable IDE0044 // Add readonly modifier
+    private CreateComputeSystemResult _result;
+#pragma warning restore IDE0044 // Add readonly modifier
```

Comment:
Compiler says these are unnecessary.
```suggestion
    private CreateComputeSystemResult _result;
```

Reply:
oh interesting, thanks will add these in